low-level/message_handlers/node_data_msg_handler.py

```diff
@@ -964,4 +1033,4 @@ def resume(self):
 
     def shutdown(self):
         """Clean up scheduler queue and gracefully shutdown thread"""
-        super(NodeDataMsgHandler, self).shutdown()
\ No newline at end of file
+        super(NodeDataMsgHandler, self).shutdown()

```

Comment:
what is the change here?

Reply:
Its a general recommendation to have a empty line at the end. I can remove it if we don't follow that.